app/src/main/java/com/jerboa/ui/components/blocks/BlocksActivity.kt

```diff
@@ -49,18 +40,13 @@ import com.jerboa.ui.components.common.LoadingBar
 import com.jerboa.ui.components.common.SimpleTopAppBar
 import com.jerboa.ui.components.common.simpleVerticalScrollbar
 import com.jerboa.ui.theme.ICON_SIZE
+import com.jerboa.ui.theme.MEDIUM_PADDING
 import com.jerboa.ui.theme.SMALL_ICON_SIZE
 import com.jerboa.ui.theme.SMALL_PADDING
 import com.jerboa.ui.theme.Title
-import it.vercruysse.lemmyapi.v0x19.datatypes.BlockCommunity
-import it.vercruysse.lemmyapi.v0x19.datatypes.BlockCommunityResponse
-import it.vercruysse.lemmyapi.v0x19.datatypes.BlockInstance
-import it.vercruysse.lemmyapi.v0x19.datatypes.BlockInstanceResponse
-import it.vercruysse.lemmyapi.v0x19.datatypes.BlockPerson
-import it.vercruysse.lemmyapi.v0x19.datatypes.BlockPersonResponse
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.launch
-import kotlinx.coroutines.withContext
+import it.vercruysse.lemmyapi.v0x19.datatypes.CommunityId
+import it.vercruysse.lemmyapi.v0x19.datatypes.Instance
+import it.vercruysse.lemmyapi.v0x19.datatypes.PersonId
 
 @Composable
 private fun BlockedElementListItem(
```

Comment:
Please move this to its own `Blocks.kt` file. Our standard has been *ideally* for the Activity to only contain the top-level screen / activity, which works with the viewmodels and does all the actions. Then another kotlin file for all the inner composables. 

Look again at a few existing folders here.